src/genetic.h

```diff
@@ -29,6 +36,6 @@ geneticAlgorithm(genetic_real* bestFitValues, const unsigned int parameterCount,
                  unsigned int generationSize, unsigned int tourneySize,
                  const unsigned int maxIterations,
                  fitness_evaluation_function function,
-                 const unsigned lowMemoryMode);
```

Comment:
You changed how you function works. Does this still works with your prototype?

Reply:
It would be better to merge this change before merging the prototype

Reply:
Hmm, I might have already merged your prototype...

Reply:
Either way is perfect thank you

Reply:
Your prototype has been merged. Please change it